concrete/src/Express/Form/Control/View/TextView.php

```diff
@@ -9,6 +9,10 @@
 class TextView extends View
 {
 
+    public function getControlID()
+    {
+        return false;
```

Comment:
This should return `null` so that when we have a PHP 7.1 minimum we can set the return type to `?string`. There is currently no supported way in PHP to set the return type to either `string` or `bool`